frontend/src/hooks/api/getters/useTrafficBundles/useTrafficBundles.ts

```diff
@@ -0,0 +1,49 @@
+import { formatApiPath } from '../../../../utils/formatPath';
+import type { ITrafficBundles } from '../../../../interfaces/trafficBundles';
+import useSWR from 'swr';
+import handleErrorResponses from '../httpErrorResponseHandler';
+import { useMemo } from 'react';
+import useUiConfig from '../useUiConfig/useUiConfig';
+
+type IUseTrafficBundlesOutput = {
+    trafficBundles: ITrafficBundles;
+    loading: boolean;
+    error?: Error;
+    refetch: () => void;
+};
+
+const defaultIncludedTraffic = (
+    isEnterprise: boolean,
+    billing?: string,
+): number => {
+    if (!isEnterprise || billing === 'pay-as-you-go') {
+        return 53;
+    }
```

Comment:
See my comments about the unit test for PAYG flag returning false

Reply:
Yeah, nice. I also realise that there's no need to memo here. useSWR should cover the needed memoization